WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -247,3 +246,26 @@ extension UIImageView {
         layer.borderColor = UIColor.clear.cgColor
     }
 }
+
+// MARK: - Logging Support
+
+/// This is just a temporary extension to try and narrow down the caused behind this issue: https://sentry.io/share/issue/3da4662c65224346bb3a731c131df13d/
+///
+private extension UIImageView {
+
+    private func logURLOptimization(from original: String, to optimized: URL) {
+        DDLogInfo("URL optimized from \(original) to \(optimized.absoluteString)")
+    }
+
+    private func logURLOptimization(from original: String, to optimized: URL, for blog: Blog) {
+        let blogInfo: String = {
+            guard blog.isAccessibleThroughWPCom() else {
+                return "self-hosted with url: \(blog.url ?? "unknown")"
+            }
+
+            return "dot-com-accessible: \(blog.url ?? "unknown"), id: \(blog.dotComID ?? 0)"
+        }()
```

Comment:
Just for fun, here's a different way to write this code that _might_ be more concise. Totally up to the author's preference. No even a nitpick, just something neat in Swift that sometimes makes it unnecessary to define an inlined closure for a conditional assignment.

```suggestion
        let blogInfo: String
        if blog.isAccessibleThroughWPCom() 
            blogInfo = "dot-com-accessible: \(blog.url ?? "unknown"), id: \(blog.dotComID ?? 0)"
        } else {
            blogInfo = "self-hosted with url: \(blog.url ?? "unknown")"
        }
```

Reply:
Yeah that was intentional.  I prefer the inline approach in general because to me it looks much cleaner and contained.

That said I don't feel strongly about either solution so I'm good with accepting your suggestion.

Reply:
The only downside about code suggestions is it's hard to know beforehand if there's any issue with it. :P

But it's in.